### PR TITLE
helm chart optimize

### PR DESCRIPTION
**Pull request by acies312**
- replace all possible properties from configmaps to app itself under kubernetes profile
- merge all files related to exact app to one yaml file
- remove git usage to migrate database
- remove mysql and s3 deployments
- remove values-images.yml from chart. values-images.yml from save-deploy will be used instead
plus minor fixes:
- set all links for cosv to gitlink
- added codehub logo svg for cosv frontend

### CODE REVIEW

**Review by akuleshov7 — APPROVED** [review]

File: api-gateway/src/main/resources/application-kubernetes.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+gateway.backend.url=http://backend
```
Comment (akuleshov7) [inline api-gateway/src/main/resources/application-kubernetes.properties:1]: Finally!

File: save-orchestrator/src/main/resources/application-kubernetes.properties [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+orchestrator.kubernetes.serviceAccount=${POD_SERVICE_ACCOUNT}
+orchestrator.kubernetes.current-namespace=${POD_NAMESPACE}
+orchestrator.kubernetes.agent-namespace=save-agent
+server.shutdown=graceful
+management.endpoints.web.exposure.include=*
+management.server.port=5101
+orchestrator.agent-settings.heartbeat-url=http://orchestrator/heartbeat
+orchestrator.agent-settings.debug=true
+logging.level.com.saveourtool.save.orchestrator.kubernetes=DEBUG
+logging.level.org.springframework=DEBUG
```
Comment (akuleshov7) [inline save-orchestrator/src/main/resources/application-kubernetes.properties:12]: I think that all properties from Spring can be moved to application.properties

Comment (nulls) [reply save-orchestrator/src/main/resources/application-kubernetes.properties:12]: it's related to k8s. better to have them here.

**Review by nulls — APPROVED** [review]

File: save-orchestrator/src/main/resources/application-kubernetes.properties (hunk 2 above)
Comment (acies312) [reply save-orchestrator/src/main/resources/application-kubernetes.properties]: done. All releted spring props moved to main properties file

**Review by Cheshiriks — APPROVED** [review]

**Pull request merged**